ideascube/static/ideascube/vendor/minislate/minislate.min.css

```diff
@@ -1,137 +0,0 @@
-.editor-toolbar {
```

Comment:
I removed it by mistake in the previous commit. I should have remove it in this one.